validation/src/test/scala/hmda/validation/rules/lar/validity/V290Spec.scala

```diff
@@ -0,0 +1,42 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+
+class V290Spec extends LarEditCheckSpec {
+
+  property("Succeeds when valid MSA/MD, state and county codes are present") {
+    forAll(larGen) { lar =>
+      whenever(lar.geography.msa != "NA") {
```

Comment:
This line seems unnecessary, as the next line sets the geography values appropriately, including `msa`.